frontend/src/context/auth.ts

```diff
@@ -1,13 +1,13 @@
-import { createContext, useContext } from 'react';
-import { ProfileObject } from './user';
-
+import { createContext, useContext } from "react";
+import firebase from "firebase";
 // These functions will be created in app.tsx
```

Comment:
Do you mean `index.tsx`? I can't seem to find the usage of `AuthContext` in `app.tsx`. Or did I overlook? 😅